### PR TITLE
Feature/687

### PR DESCRIPTION
**Pull request by gattrCh**
**Prerequisites: Always create two feature branches (frontend + backend) even if your feature only affects one of them**

**DEV:**
> Development
- [ ] Acceptance tests by dev [OpenShift feature environment]
  - [ ] Moved to "Ready for review"

> Review
- [ ] Pull-request reviewed + approved (including Code Coverage) by reviewer [GitHub]
- [ ] Inform dev after review through MEGA-PR (by reviewer) [Chat Space]
- [ ] Version upgrade by dev [GitHub Issue Milestone + maven revision]
- [ ] Merge to main by dev [GitHub]
  - [ ]  Re-add SNAPSHOT to version and push on main [maven revision]
  - [ ]  Make sure all feature branches are deleted [frontend + backend]
  - [ ]  Moved to "Ready for deployment"

> Deployment
- [ ] Integration and Unit tests on main environment passed [Workflow]
- [ ] Deployed to the test environment [GitOps Workflow triggered]
  - [ ] Moved to "Ready for testing"

**PO:**
> Testing
- [ ] Acceptance tests on the test enviroment passed
- [ ] Inspected and approved
  - [ ] Moved to "Ready for release"

### CODE REVIEW

File: src/main/java/com/gepardec/mega/service/helper/WorkingTimeUtil.java [hunk 1]
```diff
@@ -88,6 +89,12 @@ public double getOvertimeForEmployee(Employee employee,
         return (double) overtime.toMinutes() / 60;
     }
 
+    public Duration getDurationFromTimeString(String timeString) {
```
Comment (Ruhsi) [inline src/main/java/com/gepardec/mega/service/helper/WorkingTimeUtil.java:92]: @gattrCh hier mehr Sicherheitsprüfungen und entsprechende Exceptions werfen. String muss kein : enthalten; String kann leer/null sein, etc..

File: src/test/java/com/gepardec/mega/rest/ManagementResourceTest.java [hunk 2]
```diff
@@ -312,8 +317,15 @@ void getProjectManagementEntries_whenProjectTimes_thenCorrectAggregatedWorkTimes
                 ArgumentMatchers.anyString(), any(LocalDate.class)
         )).thenReturn(getProjectTimeTypeList());
 
+        when(zepService.getProjectTimes(any(Employee.class), any(LocalDate.class)))
```
Comment (Ruhsi) [inline src/test/java/com/gepardec/mega/rest/ManagementResourceTest.java:320]: @gattrCh hier dann auch null, empty, falsches format, etc tests

File: src/test/java/com/gepardec/mega/rest/ManagementResourceTest.java [hunk 3]
```diff
@@ -542,6 +559,11 @@ void getAllProjectManagementEntries_whenNoStepEntriesFound_thenReturnsResultList
         assertThat(result).isEmpty();
     }
 
+    private Duration getDurationFromTimeString(String timeString) {
```
Comment (Ruhsi) [inline src/test/java/com/gepardec/mega/rest/ManagementResourceTest.java:562]: hier entfernen und das Util verwenden

**Review by Ruhsi — CHANGES_REQUESTED** [review]

File: src/test/java/com/gepardec/mega/rest/ManagementResourceTest.java (hunk 3 above)
Comment (gattrCh) [reply src/test/java/com/gepardec/mega/rest/ManagementResourceTest.java:562]: muss hier bleiben, weil das Util gemocked verwendet wird (an anderer Stelle in der Testklasse)

**Review by Ruhsi — APPROVED** [review]

**Review by Ollitod — CHANGES_REQUESTED** [review]

**Review by Ollitod — APPROVED** [review]

**Pull request merged**